src/CalculatorButtonValues.elm

```diff
@@ -22,9 +22,12 @@ type ButtonType
 
 
 type Action
-    = Blank
-    | Add
+    = Add
+    | Blank
```

Comment:
Just going to memorialize the thoughts I related to you. I'd remove this `Blank` case entirely, since it's not an `Action`: the way you described it, it represents the absence of an `Action`. This means that you should evaluate other places in your code (e.g. your `Model`) where you expect an `Action`: do you really expect an action, or `Maybe Action`?

Reply:
replaced Blank with Maybe, thanks.